src/cascadia/TerminalControl/TermControl.h

```diff
@@ -123,7 +123,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         void _MouseTransparencyHandler(const double delta);
 
         void _ScrollbarUpdater(Windows::UI::Xaml::Controls::Primitives::ScrollBar scrollbar, const int viewTop, const int viewHeight, const int bufferSize);
-        Windows::UI::Xaml::Thickness _ParseThicknessFromPadding(const hstring padding);
+        static std::vector<double> _ParsePadding(const hstring padding);
+        static Windows::UI::Xaml::Thickness _ParseThicknessFromPadding(const hstring padding);
 

```

Comment:
Can the newly added methods be const?

Reply:
Totally missed the static declaration.  🤦‍♂  I am not sure what the "teams" guidance is here.  Should these be const member functions, static member functions or static helpers in the cpp file?  @zadjii-msft

Reply:
I had to change this to `static` because `GetProposedDimensions()` is static